synfig-studio/src/gui/dialogs/dialog_setup.cpp

```diff
@@ -76,6 +77,7 @@ Dialog_Setup::Dialog_Setup(Gtk::Window& parent):
 	adj_pref_x_size(Gtk::Adjustment::create(480,1,10000,1,10,0)),
 	adj_pref_y_size(Gtk::Adjustment::create(270,1,10000,1,10,0)),
 	adj_pref_fps(Gtk::Adjustment::create(24.0,1.0,100,0.1,1,0)),
+	adj_number_of_core(Gtk::Adjustment::create(g_get_num_processors(),1,g_get_num_processors(),1,10,0)),
```

Comment:
Reading the code of `ThreadPool::set_num_threads(int num_cores)`, the minimum value should be 2, not 1.